hera_cal/io.py

```diff
@@ -418,7 +429,7 @@ def nan_empty(shape, dtype):
     return rv
 
 
-def get_blt_slices(uvo, tried_to_reorder=False):
+def get_blt_slices(uvo, tried_to_reorder=False, assume_time_first: bool=False, assume_bl_first: bool=False):
```

Comment:
These two new arguments should be documented.

Reply:
I've removed the arguments because the new pyuvdata can check this internally now, rather than having to "assume" it.